archivist/notebooks/Sharing Album Release Info with Record Labels.ipynb

```diff
@@ -0,0 +1,235 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# RKVST_URL, RKVST_APPREG_CLIENT, RKVST_APPREG_SECRET are environment variables that represent connection parameters.\n",
+    "#\n",
+    "# RKVST_URL = represents the url to the RKVST application\n",
+    "# RKVST_APPREG_CLIENT = represents the client ID from an Application Registration\n",
+    "# RKVST_APPREG_SECRET = represents the client secret from an Application Registration\n",
+    "RKVST_URL = getenv(\"RKVST_URL\")\n",
+    "RKVST_APPREG_CLIENT = getenv(\"RKVST_APPREG_CLIENT\")\n",
+    "RKVST_APPREG_SECRET = getenv(\"RKVST_APPREG_SECRET\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "ffa23b6d-5dd5-494e-8707-6af11ea1dbe6",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Connecting to RKVST\n",
+      "RKVST_URL https://app.rkvst.io\n"
+     ]
+    }
+   ],
+   "source": [
+    "\"\"\"\n",
+    "Main function of Access Policy creation.\n",
+    "\n",
+    "* Connect to RKVST with client ID and client secret\n",
+    "* Creates an Access Policy\n",
+    "* Prints response of Access Policy creation\n",
+    "\"\"\"\n",
+    "\n",
+    "# Optional call to set the logger level.  The argument can be either\n",
+    "# \"INFO\" or \"DEBUG\".  For more sophisticated logging control see our\n",
+    "# documentation.\n",
+    "set_logger(\"INFO\")\n",
+    "\n",
+    "# Initialize connection to RKVST\n",
+    "print(\"Connecting to RKVST\")\n",
+    "print(\"RKVST_URL\", RKVST_URL)\n",
+    "arch = Archivist(RKVST_URL, (RKVST_APPREG_CLIENT, RKVST_APPREG_SECRET), max_time=300)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "bae05b9c-cafa-4e81-bcb5-3cfcf3e70de7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def create_event_access(arch):\n",
+    "    \"\"\"\n",
+    "    Creates an Access Policy that shares Album Release data for Artists with other RKVST tenancies\n",
+    "    \"\"\"\n",
+    "    props = {\n",
+    "        \"display_name\": \"Sharing Album Release Info with Record Labels\",\n",
+    "        \"description\": \"Sharing Album Release Info with Record Labels\",\n",
+    "    }\n",
+    "    filters = [{\"or\": [\"attributes.arc_display_type=Artists\"]}]\n",
+    "    access_permissions = [\n",
+    "        {\n",
+    "            \"asset_attributes_read\": [\n",
+    "                \"arc_display_name\",\n",
+    "                \"arc_display_type\",\n",
+    "                \"arc_description\",\n",
+    "            ],\n",
+    "            \"asset_attributes_write\": [],\n",
+    "            \"behaviours\": ASSET_BEHAVIOURS,\n",
+    "            \"event_arc_display_type_read\": [\"Album Release\"],\n",
+    "            \"event_arc_display_type_write\": [],\n",
+    "            \"include_attributes\": [],\n",
+    "            \"subjects\": [\"subjects/34b291c3-30f4-4d89-9ec7-85f57354f798\"],\n",
```

Comment:
How is a new user expected to get the subject ID? They will have to have done the Subject exchange dance first. Can we document this or put it in a comment cell?

Reply:
Hello @JAG-UK  . . . yep I can add some verbiage.  We do have it documented in our "docs",  however those will change but it is there.   I should probably do the same thing for the "other across tenancies" notebook (Sharing Artist Asset with Record Labels")

Reply:
Hello @JAG-UK   . . . put some comments in the first cell and repeat in the cell with the access policy function.  Didn't want to overwhelm the example with verbiage so I also put a link to documentation.  I can/will update the link as our documentation changes.